fastapi_users/db/tortoise.py

```diff
@@ -0,0 +1,54 @@
+from typing import List, Optional, Type
+
+from tortoise import Model, fields
+from tortoise.exceptions import DoesNotExist
+
+from fastapi_users.db import BaseUserDatabase
+from fastapi_users.models import BaseUserDB
+
+
+class BaseUserModel:
+    id: str = fields.TextField(pk=True, generated=False)
+    email: str = fields.CharField(index=True, unique=True, null=False, max_length=255)
+    hashed_password: str = fields.CharField(null=False, max_length=255)
+    is_active: bool = fields.BooleanField(default=True, null=False)
+    is_superuser: bool = fields.BooleanField(default=False, null=False)
```

Comment:
Unless I'm mistaken, I don't think it makes sense to have type annotation here. Those properties are of type `TextField/BooleanField/...`, not `str/bool`.

Reply:
Yes, my mistake.